test/common.jl

```diff
@@ -11,6 +11,16 @@ const DETATOL = 1e-3 * D * TRATIO
 # Random tolerance
 const RNDATOL = 5e-2 * D * TRATIO * 2
 
+# Convenience
+using Distributions: Distributions
+using Bijectors: Bijectors
+LogDensityProblems.dimension(d::Distributions.Distribution) = length(d)
+function LogDensityProblems.logdensity(d::Distributions.Distribution, y)
+    b = Bijectors.inverse(Bijectors.bijector(d))
+    x, logjac = Bijectors.with_logabsdet_jacobian(b, y)
+    return logpdf(d, x) + logjac
+end
```

Comment:
This is maybe not the greatest idea :confused: 

Should I replace this with a "local" definition of `logdensity`?

Reply:
No, I agree, I think we should remove this type piracy if possible. Where is it used?

Reply:
Cool, will do that then. It's just used a couple of places in the tests.

Reply:
Addressed it.